src/plugins/plugin.filler/index.js

```diff
@@ -65,16 +66,15 @@ export default {
   },
 
   beforeDatasetsDraw(chart, _args, options) {
-    const source = args.meta.$filler;
-
-    if (!source || source.fill === false || options.drawTime !== 'beforeDatasetsDraw') {
+    if (options.drawTime !== 'beforeDatasetsDraw') {
       return;
     }
 
     const metasets = chart.getSortedVisibleDatasetMetas();
     for (let i = metasets.length - 1; i >= 0; --i) {
       const source = metasets[i].$filler;
-      if (source) {
+      console.log(source)
```

Comment:
errant console

Reply:
I think you are looking at an old commit, that console log was already removed in: @LeeLenaleee
[remove console log and lint errors](https://github.com/chartjs/Chart.js/pull/10414/commits/82d83e7984d3522b55cfca2300046a0f86ec493c)
[82d83e7](https://github.com/chartjs/Chart.js/pull/10414/commits/82d83e7984d3522b55cfca2300046a0f86ec493c)